spec.html

```diff
@@ -26296,7 +26295,7 @@
         </emu-alg>
       </emu-clause>
 
-      <!-- es6num="20.3.1.14" -->
+      <!-- es6num="20.3.1.13" -->
```

Comment:
Just to make sure I understand you correctly before addressing this concern: 


20.3.1.8 was about DSTA which is being dropped.  And all other section number changes are due to 20.3.1.8 being dropped. 

Instead, I should just skip 20.3.1.8 and keep the old section numbers (20.3.1.9 and onward), shouldn't I?